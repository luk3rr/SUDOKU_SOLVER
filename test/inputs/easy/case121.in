302057860 000300001 000004503 080000000 009400000 260078000 506020000 700809400 000001076 
