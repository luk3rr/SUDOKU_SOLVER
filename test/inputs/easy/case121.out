Solving the following grid:
3 0 2 | 0 5 7 | 8 6 0 
0 0 0 | 3 0 0 | 0 0 1 
0 0 0 | 0 0 4 | 5 0 3 
------+-------+------
0 8 0 | 0 0 0 | 0 0 0 
0 0 9 | 4 0 0 | 0 0 0 
2 6 0 | 0 7 8 | 0 0 0 
------+-------+------
5 0 6 | 0 2 0 | 0 0 0 
7 0 0 | 8 0 9 | 4 0 0 
0 0 0 | 0 0 1 | 0 7 6 


Solution found :')

3 1 2 | 9 5 7 | 8 6 4 
6 5 4 | 3 8 2 | 7 9 1 
8 9 7 | 6 1 4 | 5 2 3 
------+-------+------
4 8 3 | 2 9 6 | 1 5 7 
1 7 9 | 4 3 5 | 6 8 2 
2 6 5 | 1 7 8 | 3 4 9 
------+-------+------
5 4 6 | 7 2 3 | 9 1 8 
7 2 1 | 8 6 9 | 4 3 5 
9 3 8 | 5 4 1 | 2 7 6 

Algorithm: BFS
Total time: 110 ms
Total expanded states: 17638
